app/models/validations/sales/sale_information_validations.rb

```diff
@@ -1,14 +1,21 @@
 module Validations::Sales::SaleInformationValidations
+  include Validations::SharedValidations
   include CollectionTimeHelper
   include MoneyFormattingHelper
 
-  def validate_practical_completion_date_before_saledate(record)
-    return if record.saledate.blank? || record.hodate.blank?
+  def validate_practical_completion_date(record)
+    return unless record.hodate.present? && date_valid?("hodate", record)
+    return if record.saledate.blank?
 
     if record.hodate > record.saledate
       record.errors.add :hodate, I18n.t("validations.sale_information.hodate.must_be_before_saledate")
       record.errors.add :saledate, I18n.t("validations.sale_information.saledate.must_be_after_hodate")
     end
+
+    if record.saledate - record.hodate >= 3.years && record.form.start_year_after_2024?
+      record.errors.add :hodate, :over_a_year_from_saledate, message: I18n.t("validations.sale_information.hodate.must_be_less_than_3_years_from_saledate")
```

Comment:
What does `over_a_year_from_saledate` give us in this scenario?

Reply:
I don't think it's doing anything. The only thing that has me slightly doubting myself is that it's also on line 30 below, and I can't see what it's doing there either.

Reply:
Looks like we used it to clear data during migrations from the old service, we don’t need it anymore!